unattended_installer/install_functions/opendistro/checks.sh

```diff
@@ -147,6 +158,38 @@ function checkArguments() {
 
 }
 
+function checkTools() {
+
+    # -------------- Check tools required to run the script (awk, sed, etc.) -----------------------------------------
+
+    toolList=(  "awk" "cat" "chown" "cp" "curl" "echo" "export"
+                "free" "grep" "kill" "mkdir" "mv" "rm" "sed"
+                "sudo" "tar" "touch" "uname")
+
+    missingtoolsList=()
+    for command in "${toolList[@]}"
+    do
+        if [ -z "$(command -v $command)" ]; then
+            missingtoolsList+=($command)
+            missingtoolsStatus="true"
+        fi
+    done
+
+    if [ -n "${missingtoolsStatus}" ]; then
```

Comment:
It's not necessary to fill the `missingtoolsStatus` variable every time that a command is not found. Remove this variable and here, check if the list contains more than 0 elements.

Reply:
Done